.circleci/config.yml

```diff
@@ -229,7 +229,9 @@ jobs:
           name: Resolve latest contracts
           working_directory: ~/project/solidity/dashboard
           command: |
-            npm upgrade @keep-network/keep-core
+            npm update @keep-network/keep-core
+            npm update @keep-network/keep-ecdsa
+            npm update @keep-network/tbtc
```

Comment:
You can run one `npm update` with multiple packages. Also, maybe use `--no-save` to avoid changing `package.json`---it's not really an issue, but I think it's conceptually cleaner (see [the docs](https://docs.npmjs.com/cli/v6/commands/npm-update#description) for more).

Reply:
Yeah, I tried with one `npm update` but I was confused with updates that happened to `package.json`. Maybe I just needed `--no-save`. Thanks for the hint!

Reply:
Actually there is one flaw of `--no-save`. It not only doesn't update `package.json` but also leaves `package-lock.json` unchanged. Taking into consideration that we're using `npm ci` that needs an updated `package-lock.json` I won't add `--no-save`.

Reply:
Some software decisions really annoy me; this is one of them :laughing: